tests/test_db.rs

```diff
@@ -251,3 +251,88 @@ fn test_sequence_number() {
         assert_eq!(db.latest_sequence_number(), 1);
     }
 }
+
+struct OperationCounts {
+    puts: usize,
+    deletes: usize,
+}
+
+impl rocksdb::WriteBatchIterator for OperationCounts {
+    fn put(&mut self, _key: Box<[u8]>, _value: Box<[u8]>) {
+        self.puts += 1;
+    }
+    fn delete(&mut self, _key: Box<[u8]>) {
+        self.deletes += 1;
+    }
+}
+
+#[test]
+fn test_get_updates_since() {
+    let path = DBPath::new("_rust_rocksdb_test_get_updates_since");
+    {
+        let db = DB::open_default(&path).unwrap();
+        //
+        // get_updates_since() on an empty database
+        //
+        let result = db.get_updates_since(0);
+        assert!(result.is_ok());
+        let mut iter = result.unwrap();
+        assert!(iter.next().is_none());
+
+        //
+        // add some records and collect sequence numbers,
+        // verify 3 batches of 1 put each were done
+        //
+        let _ = db.put(b"key1", b"value1");
```

Comment:
Here and later you're ignoring possible errors while performing `put` operation. The correct syntax would be `db.put(....).unwrap();`

Reply:
This also applies to other places where you have `let _ = `